app/src/main/res/layout/view_bottom_sheet_receive.xml

```diff
@@ -35,6 +35,9 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginTop="12dp"
+        android:gravity="center"
+        android:inputType="textMultiLine"
```

Comment:
As far as I know, the inputType is the property of EditText. It should be removed.

Reply:
Form the documentation inputType is part of TextView xml properties: https://developer.android.com/reference/android/widget/TextView#xml-attributes

Since the singleLine is deprecated devs are suggesting to use inputType: 
https://stackoverflow.com/questions/30028697/is-the-xml-attribute-singleline-deprecated-or-not-in-android/30219707

Reply:
Thanks for the detailed explanations! I checked that conversation. I've found out why it's not working without setting the `inputType`. The reason is the style named `ButtonSecondary`. It has the property `singleLine` set to true. Since this property is deprecated already, we should replace it with `maxLines="1"`. So we'll be able to get the same result without setting the `inputType`.

Reply:
@abdrasulov You have it.